Entities/Industry/CTFShops/CTFCosts.cfg

```diff
@@ -14,13 +14,15 @@
 	cost_tunnel_stone               = 100
 	cost_tunnel_wood                = 50
 	cost_tunnel_gold                = 50
+	cost_quarry_stone               = 150
+	cost_quarry_gold                = 100
```

Comment:
This commit shouldn't affect quarries, why is it there?

Reply:
Not intentional, my file must've been out of date or newer?